tests/framework/installer/ceph_installer.go

```diff
@@ -112,29 +114,31 @@ func (h *CephInstaller) CreateCephOperator(namespace string) (err error) {
 }
 
 func (h *CephInstaller) startAdmissionController(namespace string) error {
-	err := h.k8shelper.CreateNamespace(namespace)
-	if err != nil {
-		if errors.IsAlreadyExists(err) {
-			logger.Warningf("Namespace %q already exists!!!", namespace)
-		} else {
-			return fmt.Errorf("failed to create namespace %q. %v", namespace, err)
+	if h.k8sVersion >= "v1.15" {
```

Comment:
Instead of string comparison, see examples such as this for the version type:
```
k8sVersion.AtLeast(version.MustParseSemantic("v1.13.0"))
```

Reply:
Done